docs/content/en/docs/tutorials/_index.md

```diff
@@ -5,3 +5,10 @@ weight: 90
 ---
 
 See the [Github Examples page](https://github.com/GoogleContainerTools/skaffold/tree/master/examples) for examples. 
+
+To run the examples, you either have to manually replace the image repositories in the examples from gcr.io/k8s-skaffold to yours or you can point skaffold to your default image repository in one of the four ways:
```

Comment:
a little nitpicky: can this be structured something like:

```
To run the examples in the repo, you have two options:
  1) manually replace the image repositories in the example skaffold.yamls from gcr.io/k8s-skaffold to your own person repository
  2) point skaffold to your default image repository in one of the four ways:
....
```

this just makes it a bit more clear to the user why it's being done this way, and why the default-repo stuff even exists in the first place.